release.sh

```diff
@@ -68,9 +67,9 @@ npm publish --access public --tag $NPM_TAG
 cd ../../..
 
 cd packages/node || exit
```

Comment:
Should the `dht` package be released from dist folder here in the release.sh (similar to sdk)?

Reply:
It is now.